mcux/README

```diff
@@ -83,3 +83,4 @@ Patch List:
   19. drivers: fsl_sai: Allow NULL FIFO watermark
   20. drivers: fsl_sai: Fix compilation warnings when asserts are disabled
   21. mcux-sdk/drivers/ftm/fsl_ftm.c: make public FTM_GetInstance() function (mcux-sdk#155)
+  22. mcux-sdk-middleware-usb: fix ip control return value for ip3511
```

Comment:
Can you make sure there is still a newline at the end of the file here?

Reply:
newline added.

Reply:
You might need to check your editor settings- you can also verify that the change looks right with `git diff`. In the initial push, you had added a line here but had not added a newline at the end of the file in this push, you have added an additional newline. Ideally the diff would just be the addition of the line, like so:

![image](https://github.com/zephyrproject-rtos/hal_nxp/assets/34665051/1b19baf9-026c-4939-8ef6-d57924146233)

Reply:
ok now? not sure why the newline was removed in the first commit.

Reply:
Yup, looks good- no worries, and thanks :)